<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ByteMapDialog</class>
 <widget class="QDialog" name="ByteMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Byte Map</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ByteMap" name="byteMap" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="lblValue">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Value:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtValue"/>
     </item>
     <item>
      <widget class="QLabel" name="lblAddress">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtAddress"/>
     </item>
     <item>
      <widget class="QLabel" name="lblCol">
       <property name="text">
        <string>Columns</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboCols">
       <item>
        <property name="text">
         <string>128</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>256</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>512</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ByteMap</class>
   <extends>QWidget</extends>
   <header>bytemap.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ByteMapDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ByteMapDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
